tools/text_processing/split_file_on_column/split_file_on_column.xml

```diff
@@ -1,17 +1,41 @@
-<tool id="tp_split_on_column" name="Split file" version="0.3">
+<tool id="tp_split_on_column" name="Split file" version="0.2">
```

Comment:
you need to increase the version number :)

Reply:
done. increased it back to 0.3